tools/nvqpp/nvq++.in

```diff
@@ -132,6 +132,15 @@ function show_help {
 --[no-]lambda-lifting
 	Enable/disable lambda lifting pass.
 
+--opt-plugin <dynamic library file>
+	Load pass plugin by specifying its dynamic library.
+
+--opt-func-pass <pass name>
+	Append a function pass to the pass pipeline.	
+
+--opt-module-pass <pass name>
+	Append a module pass to the pass pipeline.
+
```

Comment:
Q: Can we combine these into single argument?

Reply:
We should indeed. Fixed in https://github.com/NVIDIA/cuda-quantum/pull/1034/commits/19bec7e234c865a79a5ab3028c7abaa6fbe27267.